src/components/typography.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+
+export const Bold = styled.span`
+  font-weight: 700;
+`;
+
+export const Label = props => {
+  const { text, children } = props;
+  return (
+    <Text
+      textTransform={props.textTransform || 'uppercase'}
+      letterSpacing={props.letterSpacing || '0.1rem'}
+      fontWeight='700'
+      lineHeight='168.4%;'
+      {...props}
+    >
+      {text || children}
+    </Text>
+  );
+};
+
+export const Heading = props => {
+  const { text, children } = props;
+  return (
+    <Text fontSize='2rem' variant='label' lineHeight='168.4%;'>
+      {text || children}
+    </Text>
+  );
+};
+
+export const ParaMd = props => {
+  const { text, children } = props;
+  return (
+    <Text fontSize='1rem' lineHeight='1.1rem' {...props}>
+      {text || children}
+    </Text>
+  );
+};
+export const ParaSm = props => {
+  const { text, children } = props;
+  return (
+    <Text fontSize='.9rem' lineHeight='1.1rem' {...props}>
+      {text || children}
+    </Text>
+  );
+};
+
+export const ParaLg = props => {
```

Comment:
It may be good to consolidate these Para components into a single component with a size prop

Reply:
We've tried a pretty similar system with the <TextBox /> component. It doesn't sound like a big deal, but adding a size prop (+ a variant prop in our case) turned out be a nuisance over the many text components we use.  
 
 Unless there's a big downfall of using this approach that I'm not seeing, I'd vote we try this out.